pkg/kubelet/apis/kubeletconfig/v1alpha1/types.go

```diff
@@ -246,6 +246,8 @@ type KubeletConfiguration struct {
 	FeatureGates map[string]bool `json:"featureGates,omitempty"`
 	// Tells the Kubelet to fail to start if swap is enabled on the node.
 	FailSwapOn *bool `json:"failSwapOn,omitempty"`
+	// Kubelet will reports PLEG error and be notReady if pleg relist take times greater than experimental-pleg-relist-threshold.
```

Comment:
reports -> report
take times -> takes time

Reply:
Fixed, thanks for your review.